packages/contracts-bedrock/src/libraries/Blueprint.sol

```diff
@@ -166,4 +207,8 @@ library Blueprint {
         }
         return number;
     }
+
+    function maxInitCodeSize() internal pure returns (uint32) {
```

Comment:
The original occurence of this math had the helpful comment "Preamble needs 3 bytes". Should restore that in natspec.